client/src/App.js

```diff
@@ -1,5 +1,26 @@
+import NavFooter from './components/NavFooter';
+import { Switch, Route } from 'react-router-dom';
+import Home from './pages/Home';
+import MyShelves from './pages/MyShelves';
+import MyBooks from './pages/MyBooks';
+
 function App() {
-  return <div></div>;
+  return (
+    <div>
```

Comment:
Please consider using just a React Fragment instead of a `div` as your app already is rendered in a `div` in the index.html and you don't use this extra `div` for anything.